src/sshagent/OpenSSHKey.cpp

```diff
@@ -372,10 +372,20 @@ bool OpenSSHKey::openPrivateKey(const QString& passphrase)
             return false;
         }
 
-        QCryptographicHash hash(QCryptographicHash::Md5);
-        hash.addData(passphrase.toUtf8());
-        hash.addData(m_cipherIV.data(), 8);
-        QByteArray keyData = hash.result();
+        QByteArray keyData;
+        QByteArray mdBuf;
+        for (;;) {
```

Comment:
You should use a do/while loop instead.

Reply:
That's really nitpicky.

Reply:
Maybe, but it's ensuring we are keeping a consistent style. And TBH, an empty for loop header is just extremely ugly.

Reply:
Changed, but I'm not sure it looks any better.

Reply:
It does. Thank you very much.